sdk/core/Azure.Core.Experimental/src/JsonDataChange.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Azure.Core.Dynamic
+{
+    internal struct JsonDataChange<T> : IJsonDataChange
```

Comment:
Why did you make change record to be generic? Is it to avoid boxing the T value? If yes, then I think you should go back to doing it like in the prototype, i.e. use Value type (which will need to be productized too). The reason is that I don't think you can make the changelist non-boxing with generics. Generics force you to add the interface, and then then in turn casing a struct to an interface will box the struct. The Value type can store all primitive value types without boxing.

Reply:
Is there a way to accomplish the changelist without productizing the `Value` type?  I don't want JsonData to require a dependency on this.